python/pyspark/sql/functions.py

```diff
@@ -6158,12 +6764,14 @@ def _test() -> None:
     import doctest
     from pyspark.sql import Row, SparkSession
     import pyspark.sql.functions
+    import math
 
     globs = pyspark.sql.functions.__dict__.copy()
     spark = SparkSession.builder.master("local[4]").appName("sql.functions tests").getOrCreate()
     sc = spark.sparkContext
     globs["sc"] = sc
     globs["spark"] = spark
+    globs["math"] = math
```

Comment:
Let's import this within the docstring. I think we should ideally remove all these globals (except `spark`).

Reply:
removed, I can look into removal of "sc" in the next PRs